scripts/import_bch_flat.py

```diff
@@ -32,14 +35,14 @@
     session = Session()
     log.info('Database connected')
 
-    for line in fileinput.input(args.args):
+    for line in fileinput.input(args.filelist):
         fp = line.strip()
         if not fp or fp.startswith('#'):
             continue
         log.info('Processing {}'.format(fp))
         with open(fp, 'rb') as f:
             data = csv.DictReader(f)
-            r = bch.process(data, session, args.diag)
+            r = bch.process(data, session, args.diag, int(args.batch_size))
             log.info('Done file: {}'.format(r))
 
     session.commit()
```

Comment:
Hmm, I doubt this commit is necessary, given that all transactions in this session are nested (and either committed or rolled back). 

Also, there should be a `session.close()` here to end the whole thing; it's part of the standard session lifecycle (although I know our code often misses this nicety). See [SQLAlchemy Session Basics](https://docs.sqlalchemy.org/en/latest/orm/session_api.html); search for `session.close()`.